bagua/torch_api/contrib/cache_loader.py

```diff
@@ -0,0 +1,135 @@
+import pickle
+from collections import defaultdict
+
+__all__ = ["CacheLoader"]
+
+
+def serialize(input):
+    return pickle.dumps(input)
+
+
+def deserialize(input):
+    return pickle.loads(input)
+
+
+class CacheLoader:
```

Comment:
cachedloader

Reply:
cacheloader is named after guava cache loader

however, if you insist, i'll change it.

Reply:
cached loader should be more concise

Reply:
oh this is not a pytorch dataloader. Then keep it `CacheLoader`